### ISSUE TITLE
About Handling Universal Links

### ISSUE THREAD

**Issue by Steven-Guo** [RoverPlatform/rover-ios#139]
Hi Sean,

The documentation said use Rover.continueUserActivity(userActivity) and Rover.open(url: url) to handle deep link, but it seems that the SDK doesn't have these methods. Any update on this?

Thanks,
Steven

**Comment by seanrucker** [comment]
Do you have the latest version of the SDK (1.6.0)?

https://github.com/RoverPlatform/rover-ios/blob/v1.6.0/Rover/Rover.swift#L256

**Comment by Steven-Guo** [comment]
My pod seems still grab 1.5.0, let me try again.

**Comment by Steven-Guo** [comment]
Problem solved, thank you.
